src/osal/lv_os.h

```diff
@@ -143,6 +143,28 @@ lv_result_t lv_thread_sync_signal(lv_thread_sync_t * sync);
  */
 lv_result_t lv_thread_sync_delete(lv_thread_sync_t * sync);
 
+/**
+ * Lock LVGL's general mutex.
+ * LVGL is not thread safe, so a mutex is used to avoid executing multiple LVGL functions at the same time
+ * from different threads. It shall be called when calling LVGL functions from threads
+ * different than lv_timer_handler's thread. It doesn't need to be called in LVGL events because
+ * they are called from lv_timer_handler().
+ * It is called internally in lv_timer_handler().
+ */
+void lv_lock(void);
+
+/**
+ * Same as `lv_lock()` but can be called from an interrupt.
+ */
+void lv_lock_isr(void);
```

Comment:
```suggestion
lv_result_t lv_lock_isr(void);
```

Please see my review.

Reply:
Okay, done!